templates/repo/branch/list.tmpl

```diff
@@ -110,12 +110,12 @@
 								{{if and (not .DBBranch.IsDeleted) $.DefaultBranchBranch}}
 								<div class="commit-divergence">
 									<div class="bar-group">
-										<div class="count count-behind">{{.CommitsBehind}}</div>
+										<div class="count count-behind" data-tooltip-content="{{if eq .CommitsBehind 1}}{{ctx.Locale.Tr "repo.branch.commits_behind_1" .CommitsBehind $.DefaultBranchBranch.DBBranch.Name}}{{else}}{{ctx.Locale.Tr "repo.branch.commits_behind_n" .CommitsBehind $.DefaultBranchBranch.DBBranch.Name}}{{end}}">{{.CommitsBehind}}</div>
```

Comment:
Can . CommitsBehind be zero?

Reply:
Yes
![image](https://github.com/user-attachments/assets/6b9f2b95-6e1b-46c6-83b7-dcd2ce2a70aa)
![image](https://github.com/user-attachments/assets/4eaf248e-5f1e-4739-9a5d-78947aec8be7)

Reply:
I am struggling with this, because the existing keys for `commit` and `commits` are uppercase. Do I add a new key like `commit_lowercase` specifically for this?